src/BreakfastSerializer/Plugin/IsPlugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace BDBStudios\BreakfastSerializer\Plugin;
+
+/**
+ * Interface IsPlugin
+ * @package BDBStudios\BreakfastSerializer\Plugin
+ */
+interface IsPlugin
```

Comment:
Noted, always felt that was a bit redundant, like calling a thing Thing, will update